backend/capellacollab/projects/routes.py

```diff
@@ -89,12 +89,7 @@ def get_project_by_slug(
     return db_project
 
 
-@router.post(
-    "/",
-    response_model=Project,
-    tags=["Projects"],
-    dependencies=[Depends(RoleVerification(required_role=Role.ADMIN))],
-)
+@router.post("/", response_model=Project, tags=["Projects"])
```

Comment:
```suggestion
@router.post(
    "/",
    response_model=Project,
    tags=["Projects"],
    dependencies=[Depends(RoleVerification(required_role=Role.USER))],
)
```

The user has to be authenticated, so a check for the role is still required.